### PR TITLE
Update terms documentation to include publishing features.

### PR DESCRIPTION
**Pull request by cmars**

### CODE REVIEW

File: src/en/developer-terms.md [hunk 1]
```diff
@@ -20,11 +20,62 @@ order to deploy.
 
 ## Creating terms
 
-At the moment, terms can only be created by charm store administrators.
-Charm developers should send an email to juju@lists.ubuntu.com with:
-- the id of the term in the format of "vendor-app", such as "lorem-ipsum".
-- the full text of the terms to display to the user, appended to the body
-of the email.
+As a charm developer, you may push and release terms owned by a group of which
+you are a member. Groups are mapped onto Launchpad teams for the time being.
```
Comment (evilnick) [inline src/en/developer-terms.md]: for the time being? Is this going to change? Maybe "At present, groups are mapped onto Launchpad teams, though in future..."

Comment (cmars) [inline src/en/developer-terms.md]: I can remove this to reflect current reality.

Comment (evilnick) [inline src/en/developer-terms.md]: ok

**Comment by cmars** [comment]
@evilnick Updated, PTAL, thanks!

**Pull request merged**